### PR TITLE
LMS-1525-6 | DR changes

### PR DESCRIPTION
**Pull request by preziu**

### CODE REVIEW

File: lib/js/components/PopOver/PopOver.vue [hunk 1]
```diff
@@ -51,6 +51,7 @@
 @import '../../../styles/settings/radiuses';
 @import '../../../styles/settings/shadows';
 @import '../../../styles/settings/spacings';
+@import '../../../styles/settings/media-queries';
```
Comment (kvas-damian) [inline lib/js/components/PopOver/PopOver.vue:54]: is it required?

**Review by kvas-damian — APPROVED** [review]

File: lib/js/components/PopOver/PopOver.vue (hunk 1 above)
Comment (preziu) [reply lib/js/components/PopOver/PopOver.vue]: Leftover from tests, deleted.

File: lib/js/components/PopOver/PopOver.vue [hunk 2]
```diff
@@ -70,8 +70,13 @@
 		background-color: $color-neutral-background;
 	}
 
+	.popOver__contentSlot {
+		word-break: break-word;
+	}
+
 	.popOver__contentSlot.-maxHeight {
 		overflow: scroll;
+		overflow-x: hidden;
```
Comment (kvas-damian) [inline lib/js/components/PopOver/PopOver.vue:79]:
```suggestion
		overflow: hidden scroll;
```

**Pull request merged**